charts/celeborn/Chart.yaml

```diff
@@ -39,4 +39,4 @@ version: 0.1.0
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 0.2.1
+appVersion: 0.4.0
```

Comment:
So this version should be the same with Celeborn release version, right?

Reply:
As metioned by the above:

> # This is the version number of the application being deployed. This version number should be
# incremented each time you make changes to the application. Versions are not expected to
# follow Semantic Versioning. They should reflect the version the application is using.

We should update this when we release celeborn.